holidays/locale/en_US/LC_MESSAGES/SK.po

```diff
@@ -17,86 +17,86 @@ msgstr ""
 "X-Generator: Poedit 3.2.2\n"
 
 #. 100th anniversary of the adoption of the Declaration of the Slovak Nation.
```

Comment:
Avoid including changes to files that are not relevant to PR.

Reply:
this file are know why change, but you right. my bad.